happypose_ros/happypose_ros/happypose_node.py

```diff
@@ -631,6 +642,26 @@ def _await_results(self, cam_data: dict) -> None:
                     results, header, has_bbox=not self._multiview
                 )
 
+                ##################
+                # for contact_graspnet_ros
+                masks = results["masks"].numpy()
+                merged_mask = np.zeros(masks[0].shape, dtype=np.uint16)
+                for instance_id in range(len(masks)):
+                    # label format: tless-obj_000021
+                    label = results["infos"].iloc[instance_id].label
+                    label_id = int(label[-6:])
+                    seg_id = pairing.pair(
```

Comment:
I know it's already late on this PR, so this comment can be ignored. I think maybe a better idea would be publish a message containing those pairs as a mapping. This implementation is ok, tho it is not very standard and obvious on how to revert this transformation

Reply:
`label_id, instance_id = label_pairing.depair(seg_id)`
The only downside is an added dependency for the client.